mlrun/api/api/endpoints/submit.py

```diff
@@ -84,6 +84,9 @@ async def submit_job(
             labels.setdefault("v3io_user", username)
             labels.setdefault("owner", username)
 
+    data["task"]["metadata"]["labels"].update(
+        {"client_version": data.headers.get("x-mlrun-client-version")}
```

Comment:
IIRC we have some conventions for our labels, prefix is `mlrun/` or something like that, check that